core/transaction/transactionSorter.go

```diff
@@ -0,0 +1,42 @@
+package transaction
+
+import (
+	"bytes"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"sort"
+)
+
+func SortTransactionsBySenderAndNonceWithFrontRunningProtection(transactions []data.TransactionHandler, hasher hashing.Hasher, randomness []byte) {
+	// make sure randomness is 32bytes and uniform
+	randSeed := hasher.Compute(string(randomness))
+	xoredAddresses := make(map[string][]byte)
+
+	for _, tx := range transactions {
```

Comment:
will there be a commit in elrond-go that will call this function? or will the function remain duplicated there as well?

Reply:
It won't just work as a replaceable drop-in because the elrond-go`s verion uses txcache.WrappedTransaction instead of TransactionHandler

Reply:
yes, it will be used instead of the moved code, not duplicate
https://github.com/ElrondNetwork/elrond-go/pull/4712/files